LanguageSpec.md

```diff
@@ -116,13 +116,17 @@ So it would be:
 
 ### Place Character
 
-If we wanted to move the character sprite anywhere on the screen, we could just type
+If we wanted to move the character sprite anywhere on the screen with scaling, we could just type
 
-```PLACE [Character] ([Up, Down, Left, Right])```
+```PLACE [Character] ([x, y, Scale-Height, Scale-Width])```
 
```

Comment:
is the scaling optional or will the user have to supply scale every time? 
If it's every time, that's increases the amount of typing the user has to do for every PLACE command.

Reply:
Never mind, should've read on first.